Formula/qlik-cli.rb

```diff
@@ -0,0 +1,26 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+class QlikCli < Formula
+  desc "qlik-cli is a CLI for using Qlik Sense Cloud editions."
+  homepage "https://github.com/qlik-trial/qlik-cli"
```

Comment:
Is this required?

Reply:
I think so!

Reply:
Ok, then I suggest we put the homebrew-taps repo as homepage since, the private repo will not be accessible externally.

Reply:
It wasn't needed, turns out.